src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderMovementsResourceIntTest_cancelCourtHearing.java

```diff
@@ -131,7 +131,7 @@ private Collection<CourtHearing> givenAllCourtHearings(final String token) {
                 }).getBody();
 
         assertThat(courtHearings).isNotNull();
-        assertThat(courtHearings.getHearings().size()).isGreaterThanOrEqualTo(1);
+        assertThat(courtHearings.getHearings()).hasSizeGreaterThanOrEqualTo(1);
```

Comment:
just made the test a bit nicer as it was failing with a NullPointer which wasn't helpful